Porting/release_managers_guide.pod

```diff
@@ -1216,6 +1224,9 @@ Use the template at Porting/release_announcement_template.txt
 
 Send a carbon copy to C<noc@metacpan.org>
 
+If your email does not appear on the list, but does not obviously bounce
+either, check that the email you are sending from is subscribed to the list.
+
```

Comment:
As above

Reply:
@tonycoz, could you and @richardleach work out what happened/what is needed -- because I think that is the only obstacle to merging this ticket.